### DISCUSSION TITLE
create a new solid

### DISCUSSION THREAD

**Discussion by normandajc** [dune3d/dune3d#141] [General]
In the example case.d3ddoc, you have two solids, main and lid. How do I create a new solid?

**Comment by carrotIndustries** [comment]
See https://docs.dune3d.org/en/latest/groups.html#groups-and-bodies

> To add a body, select the checkbox next to “Body” in the Group tab.

**Comment by carrotIndustries** [comment]
Since you're not the first to ask this: Where did you look but not find it?

**Comment by normandajc** [comment]
Thanks for the answer.
I have created a new body but it imposes chamfer and I have no reference.
![newBody](https://github.com/user-attachments/assets/55d73316-5050-45bf-958d-80c895099d60)

**Comment by carrotIndustries** [comment]
> I have created a new body but it imposes chamfer and I have no reference.

Selecting the body checkbox just makes the selected group the start of a body. You probably want to create a new sketch group first.

To simplify this step, 7b1dd707120e609479acdfb5edf09d0b565b33f0 adds "Sketch in new Body" to the add group menu.
